server/Model/meeting.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const meetingSchema = new Schema(
+  {
+    user_id: {
+      type: Schema.Types.ObjectId,
+      ref: "User",
+      required: true,
+    },
+    learning_id: {
```

Comment:
I don't think we should restrict a meeting to be connected to a learning. If anything it should be connected to one of the experts expertise.

Reply:
> I don't think we should restrict a meeting to be connected to a learning. If anything it should be connected to one of the experts expertise.

`learning_id` is something optional. Added it so if in future we want to have a feature that expert can approach a learner, it will be helpful in that case.